tests/test-addons.py

```diff
@@ -57,7 +63,22 @@ def test_help_text(self):
         expected["ha-cluster"] = "enabled"
         expected["helm"] = "enabled"
         expected["helm3"] = "enabled"
-        expected["dns"] = "enabled"
+
+        # Check MicroK8s version
+        cmd = "/snap/bin/microk8s.version"
+        version_str = run_until_success(cmd, timeout_insec=300)
+        version = check_output(
+            f"echo {version_str} | sudo grep -oP '\\d+\\.\\d+' ",
+            shell=True,
+            encoding="utf-8",
+        ).strip()
+        minor_version = int(version[2:])
+
+        # dns is enabled by default in versions 1.27+
```

Comment:
I am not sure what is the point of this given the master branch of the addons repository is used on the 1.27 (for now) MicroK8s only. If we want to handle the test behavior on pre-1.27 releases we should be patching the respective branches (eg 1.26).

Reply:
Okay removing from this PR.